examples/chef/chef_config.py

```diff
@@ -0,0 +1,18 @@
+import os
+#
+# USER DEFINED INFORMATION
```

Comment:
Why are we doing configuration by editing an existing file? This will dirty the tree.

Reply:
fixed, it wasn't needed